wa/commands/show.py

```diff
@@ -91,13 +91,7 @@ def get_rst_from_target(target):
     if hasattr(target, 'description'):
         desc = strip_inlined_text(target.description or '')
         text += desc
-    conn_params_rst = get_params_rst(target.conn_params)
-    if conn_params_rst:
-        text += underline('\nconnection parameters', '~') + conn_params_rst
-    platform_params_rst = get_params_rst(target.platform_params)
-    if platform_params_rst:
-        text += underline('\nplatform parameters', '~') + platform_params_rst
-    target_params_rst = get_params_rst(target.target_params)
-    if target_params_rst:
-        text += underline('\nconnection parameters', '~') + target_params_rst
+    text += get_params_rst(target.conn_params)
```

Comment:
I's probably worth having a single heading for all three, e.g. "device parameters" to reflect the name of the agenda section where they go.

Reply:
Updated